chrome/manifest.json

```diff
@@ -22,7 +22,9 @@
     "cookies",
     "declarativeNetRequestWithHostAccess",
     "webRequest",
-    "storage"
+    "storage",
+	"contextMenus",
```

Comment:
This indentation difference makes it look like you didn't run `npm run format`, so please confirm that and `npm test` passes.

Reply:
Hi there, good point - you are correct. I did indeed forget to run npm test.

There is another change that's needed based on npm test - the `let` variables should be `const`s.

I'll get to this over the weekend alternatively if you prefer to do it sooner I don't mind if someone from Kagi team does it either - all good either way.

Thanks for the guidance!

Reply:
I'll leave you to it, it's not urgent. :blush: